cmd/nexd/main.go

```diff
@@ -5,24 +5,32 @@ import (
 	"fmt"
 	"os"
 	"os/signal"
+	"runtime"
 	"sort"
 	"sync"
 	"syscall"
 
 	"github.com/nexodus-io/nexodus/internal/nexodus"
+	"github.com/nexodus-io/nexodus/internal/util"
 	"github.com/urfave/cli/v2"
 	"go.uber.org/zap"
 )
 
 const (
-	nexodusLogEnv = "NEXD_LOGLEVEL"
+	nexodusLogEnv    = "NEXD_LOGLEVEL"
+	Linux            = "linux"
```

Comment:
Instead of redefining this, I would use the `Linux` constant already defined in the `nexodus` package.